src/containers/chat/chat.js

```diff
@@ -5,22 +5,30 @@ import MessengerCustomerChat from 'react-messenger-customer-chat';
 import { useSelector } from 'react-redux';
 import { config } from 'react-spring';
 import { Transition } from 'react-spring/renderprops';
+import { useQuery } from '@apollo/client';
+import { getContactsForChat } from './operations/chat-contacts-query';
+import errorOrLoadingHandler from '../../utils/errorOrLoadingHandler';
 import { useStyles } from './chat.style';
 import MailForm from './mail-form';
 import { CHAT_FACEBOOK_DATA } from '../../configs/index';
 
 export const Chat = () => {
   const [iconsVisible, setIconsVisible] = useState(false);
   const [mailFormVisible, setMailFormVisible] = useState(false);
-  const { language, themeMode, contacts } = useSelector((state) => ({
+  const { language, themeMode } = useSelector((state) => ({
     language: state.Language.language,
-    themeMode: state.Theme.lightMode,
-    contacts: state.Contacts.contacts
+    themeMode: state.Theme.lightMode
   }));
 
   const style = useStyles({ themeMode, iconsVisible, mailFormVisible });
   const cancelIconHandler = () => setMailFormVisible(!mailFormVisible);
 
+  const [contacts, setContacts] = useState([]);
```

Comment:
Move useState to top of the function to all useStates.

Reply:
Vladimir Romanovsky said that this problem should be fixed. He is working of the chat component. I removed this from the component.